tests/installation/installation_mode.pm

```diff
@@ -16,8 +16,10 @@ sub run() {
     }
 
     if ( get_var("ADDONURL") || get_var("ADDONS") ) {
-        send_key "alt-c";    # Include Add-On Products
-        assert_screen "addonproduct-included", 10;
+        if (!check_var('ADDONS', 'smt')) {  # no include for SMT add-on bnc928895
```

Comment:
this will bite you at some point. I suggest you add some function to check $ADDONS in the basetest to call as $self->is_addon_registered('smt')

Next time you're asked to test smt+sdk and your code will do the wrong thing